components/UnifiedDocFeed/utils/getSelectedUrlFilters.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  feedTypeOpts,
+  sortOpts,
+  tagFilters,
+  topLevelFilters,
+} from "../constants/UnifiedDocFilters";
+import { scopeOptions } from "~/config/utils/options";
+
+export type SelectedUrlFilters = {
+  topLevel: string | undefined;
+  type: string | undefined;
+  sort: string | undefined;
+  time: string | undefined;
```

Comment:
Please use `NullableString` unless we wanted to make use of `undefined` specifically

Reply:
Cool. Didn't really know about this type